test/support/fake_transaction.ex

```diff
@@ -34,7 +34,8 @@ defmodule Appsignal.FakeTransaction do
     Agent.get(__MODULE__, &Map.get(&1, :finished_events, []))
   end
 
-  def try_set_action(_transaction, conn) do
+  def try_set_action(_transaction, conn), do: try_set_action(conn)
+  def try_set_action(conn) do
     try do
       value = "#{conn.private.phoenix_controller}##{conn.private.phoenix_action}"
```

Comment:
i see that in the phoenix log the controller action combination is logged as:

```
AppsignalPhoenixExample.PageController.exception/2
```

now it shows up like how we display Ruby controllers:

```
AppsignalPhoenixExample.PageController#exception
```

could we show it in the same format?